core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1259,66 +1268,86 @@
   <xs:complexType name="memory-object">
     <xs:attribute name="size" type="xs:long" default="-1">
       <xs:annotation>
-        <xs:documentation>The size of the eviction cache as a long.  Limits the cache to this normal by the amount of
-        entries in the cache.</xs:documentation>
+        <xs:documentation>
+          Defines the size of the data container as a long. Eviction occurs
+          when the number of entries exceeds the size.
+        </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="strategy" type="tns:eviction-strategy">
       <xs:annotation>
-        <xs:documentation>The eviction strategy to use. This determines if eviction is even enabled or if it
-          has a different variant.</xs:documentation>
+        <xs:documentation>
+          Specifies a strategy for evicting cache entries. The default strategy
+          is NONE so you must specify a strategy for eviction to take place.
```

Comment:
This isn't true. If a size is supplied and it is NONE or MANUAL we override to REMOVE and log a message.

Reply:
@wburns  so `size=>0` and `strategy=NONE` then `REMOVE`

also if `size=>0` and `strategy=MANUAL` then `REMOVE`? that sounds like manual eviction doesn't work.

Reply:
MANUAL always works - no matter the setting. Manual is used solely for the purpose of not having a warning message when you start the cache and have passivation without eviction being explicitly enabled.

We are trying to help the user out, if they are setting a size they most likely want an eviction type, we don't want to block that (usually for the NONE case).